<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(175, 212, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="choco">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>220</y>
      <width>521</width>
      <height>651</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Uroob</family>
      <pointsize>56</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/Screenshot_from_2024-07-04_12-20-15-removebg-preview.png);background-color: rgba(0, 0, 0, 0);border: none; </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="strawberry">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>240</y>
      <width>500</width>
      <height>611</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Uroob</family>
      <pointsize>56</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/Screenshot_from_2024-07-04_12-19-44-removebg-preview.png);background-color: rgba(0, 0, 0, 0);border: none; </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mint">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>240</y>
      <width>440</width>
      <height>610</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Uroob</family>
      <pointsize>56</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/Screenshot_from_2024-07-04_12-20-02-removebg-preview.png);background-color: rgba(0, 0, 0, 0);border: none; </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="kiosklabel_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>491</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dyuthi</family>
      <pointsize>60</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Pick Topping !</string>
    </property>
   </widget>
   <widget class="QPushButton" name="oreoBtn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>390</y>
      <width>121</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Uroob</family>
      <pointsize>56</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/Screenshot_from_2024-07-04_13-22-28-removebg-preview.png);background-color: rgba(0, 0, 0, 0); /* 투명 배경 */border: none; /* 버튼 테두리 제거 */</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="chocoballBtn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>400</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Uroob</family>
      <pointsize>56</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/Screenshot_from_2024-07-04_13-22-43-removebg-preview.png);background-color: rgba(0, 0, 0, 0); /* 투명 배경 */border: none; /* 버튼 테두리 제거 */</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cerialBtn">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>400</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Uroob</family>
      <pointsize>56</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/Screenshot_from_2024-07-04_13-22-47-removebg-preview.png);background-color: rgba(0, 0, 0, 0); /* 투명 배경 */border: none; /* 버튼 테두리 제거 */</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="chocotopping">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>240</y>
      <width>571</width>
      <height>621</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Uroob</family>
      <pointsize>56</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
image: url(:/icon/Screenshot_from_2024-07-04_13-20-28-removebg-preview.png);
background-color: rgba(0, 0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cerialtopping">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>280</y>
      <width>431</width>
      <height>521</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Uroob</family>
      <pointsize>56</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/Screenshot_from_2024-07-04_13-20-51-removebg-preview.png);
background-color: rgba(0, 0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="oreotopping">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>260</y>
      <width>641</width>
      <height>591</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Uroob</family>
      <pointsize>56</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/Screenshot_from_2024-07-04_13-21-06-removebg-preview.png);
background-color: rgba(0, 0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="kiosklabel_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>510</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dyuthi</family>
      <pointsize>30</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Oreo</string>
    </property>
   </widget>
   <widget class="QLabel" name="kiosklabel_5">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>510</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dyuthi</family>
      <pointsize>30</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Choco</string>
    </property>
   </widget>
   <widget class="QLabel" name="kiosklabel_6">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>510</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dyuthi</family>
      <pointsize>30</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Cerial</string>
    </property>
   </widget>
   <widget class="QLabel" name="turkey">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>160</y>
      <width>340</width>
      <height>180</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Uroob</family>
      <pointsize>56</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/free-icon-turkey-12694430.png);
background-color: rgba(191, 64, 64, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="wait">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-110</y>
      <width>1920</width>
      <height>900</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/icon/Screenshot from 2024-07-04 12-05-16.png);
    background-position: bottom center; /* 바닥 중앙에서 시작 */
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="orderlabel">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>530</y>
      <width>281</width>
      <height>110</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Waree</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Let's Go</string>
    </property>
   </widget>
   <widget class="QPushButton" name="infoBtn">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>490</y>
      <width>301</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0); /* 투명 배경 */
    border: none; /* 버튼 테두리 제거 */
    image: url(:/icon/free-icon-direction-1653447.png); /* 아이콘 설정 */</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>wait</zorder>
   <zorder>choco</zorder>
   <zorder>strawberry</zorder>
   <zorder>mint</zorder>
   <zorder>kiosklabel_3</zorder>
   <zorder>chocotopping</zorder>
   <zorder>cerialtopping</zorder>
   <zorder>oreotopping</zorder>
   <zorder>kiosklabel_4</zorder>
   <zorder>kiosklabel_5</zorder>
   <zorder>kiosklabel_6</zorder>
   <zorder>turkey</zorder>
   <zorder>oreoBtn</zorder>
   <zorder>chocoballBtn</zorder>
   <zorder>cerialBtn</zorder>
   <zorder>infoBtn</zorder>
   <zorder>orderlabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource_topping.qrc"/>
 </resources>
 <connections/>
</ui>
